plugins/CoreAdminHome/CustomLogo.php

```diff
@@ -50,7 +50,7 @@ public function getSVGLogoUrl($pathOnly = false)
 
     public function isEnabled()
     {
-        return (bool) Option::get('branding_use_custom_logo');
+        return $this->isCustomLogoFeatureEnabled() && (bool) Option::get('branding_use_custom_logo');
```

Comment:
The `&&` operation automatically casts the result to boolean, so we can remove the explicit casting.
```suggestion
        return $this->isCustomLogoFeatureEnabled() && Option::get('branding_use_custom_logo');
```